Cargo.toml

```diff
@@ -1,5 +1,6 @@
 [workspace]
 members = [
+    "matchbox_common",
```

Comment:
I think `matchbox_protocol` would probably be a better name? this is why I asked if there is any other code we should share... I couldn't come up with any.

Reply:
`matchbox_protocol` seems like a good name to me

Reply:
I expect we would make good use of it if we decided to serialize Signals in multiple formats

Reply:
renamed to `matchbox_protocol`